docs/sites.yml

```diff
@@ -5292,3 +5292,14 @@
   built_by: NGWIN
   built_by_url: https://picpick.app/
   featured: false
+- title: Ste O'Neill
+  main_url: https://www.steoneill.dev
+  url: https://www.steoneill.dev
+  description: >
+    MVP of a portfolio site for a full stack UK based developer.
+  categories:
+    - Blog
```

Comment:
Is the blog a work in progress?

Reply:
It is indeed! 

It'll be pushed in a week or so